### PR TITLE
New Blog Post: Introducing the GraphQL Add-on

### PR DESCRIPTION
**Pull request by ricekot**
- Add new blog post titled, "Introducing the GraphQL Add-on for ZAP"

Signed-off-by: ricekot <ricekot@gmail.com>

### CODE REVIEW

**Review by psiinon — APPROVED** [review]
Looks good to me :)

File: site/content/blog/2020-08-28-introducing-the-graphql-add-on-for-zap/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+- PR [#2420](https://github.com/zaproxy/zap-extensions/pull/2420) - Import Dialog
+- PR [#2431](https://github.com/zaproxy/zap-extensions/pull/2431) - Import Functionality
+- PR [#2485](https://github.com/zaproxy/zap-extensions/pull/2485) - Generate GraphQL Queries from Schema
+- PR [#2521](https://github.com/zaproxy/zap-extensions/pull/2521) - Add Option for Separated Arguments
+- PR [#6044](https://github.com/zaproxy/zaproxy/pull/6044) - Call default spider filter after custom filters
+
+The released add-on package can be found [here](https://github.com/zaproxy/zap-extensions/releases/tag/graphql-v0.1.0).
+
+If you'd like to know more about the implementation details of this add-on, you can go to the [blog](https://akshathkothari.com/gsoc/) for this project. If you are also interested in contributing to ZAP, I urge you to read the "[Hacking ZAP](https://www.zaproxy.org/blog/2014-03-10-hacking-zap-1-why-should-you/)" series of blog posts or go through the recently released "[ZAP in Ten](https://www.alldaydevops.com/zap-in-ten)" videos. Finally, go check out [CONTRIBUTING.md](https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md).
```
Comment (thc202) [inline site/content/blog/2020-08-28-introducing-the-graphql-add-on-for-zap/index.md:108]: `https://www.zaproxy.org` could be removed to keep links relative, when testing locally.

File: site/content/blog/2020-08-28-introducing-the-graphql-add-on-for-zap/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+Now, if you go to *Tools -> Options -> GraphQL*, you will find the following settings panel:
+
+![The GraphQL Options Panel](images/graphql-options-panel.png)
+
+These settings allow you to control the output of the add-on's Query Generator. Based on the combination of your chosen values for the above options, you will be able to send unique queries to the endpoint.
+
+For example, by increasing the maximum query depth, you may be able to exhaust the server's resources. Or if you specify arguments using variables, you could perform injection attacks by making use of ZAP's input vectors. You may also be able to bypass authorization and discover IDORs by generating queries with or without certain arguments and fields. The possibilities are endless!
+
+I did say before, that the add-on is still in an early stage. If you face any problems or have any feature requests, feel free to join in the discussion on the [ZAP User Group](https://groups.google.com/d/msg/zaproxy-users/j0Uxg6cUIY8/ghBP26sjBgAJ) or contact me directly. You can also ping us on the irc channel ([#zaproxy on Freenode](https://webchat.freenode.net/#zaproxy)) or raise a GitHub issue for bugs.
```
Comment (thc202) [inline site/content/blog/2020-08-28-introducing-the-graphql-add-on-for-zap/index.md:91]: This link is to an unrelated topic/message, was it meant as `https://groups.google.com/group/zaproxy-users`?

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thanks!

**Pull request merged**